static/js/util/profile_edit_test.js

```diff
@@ -0,0 +1,112 @@
+import assert from 'assert';
+import moment from 'moment';
+
+import {
+  boundTextField,
+  boundDateField,
+  boundSelectField
+} from './profile_edit';
+
+describe('Profile Editing utility functions', () => {
+  let that;
+  const change = (clone) => that.props.profile = clone;
+  beforeEach(() => {
+    that = {
+      props: {
+        profile: {
+          "first_name": "",
+          "date_of_birth": "",
+          "gender": undefined,
+        },
+        errors: {},
```

Comment:
Can you add an error here (or below) so that we can assert it in the return value?